src/containers/Chat/ChatMessages/AddVariables/AddVariables.tsx

```diff
@@ -25,29 +28,67 @@ export const AddVariables: React.FC<AddVariablesPropTypes> = ({
   variableParam,
 }: AddVariablesPropTypes) => {
   const [formFieldItems, setFormFieldItems] = useState<any>([]);
-  const [validation, setValidation] = useState<any>({});
   const [initialValues, setInitialValues] = useState<any>({});
   const { t } = useTranslation();
+  const [contactVariables, setContactVariables] = useState<any>([]);
+
+  let contacts: any = [];
+  let fields: any = [];
+  let selectedVariables: any = [];
+
+  const glificBase = FLOW_EDITOR_API;
+  const prefix1 = '@contact.fields.';
```

Comment:
```suggestion
  const contactFieldsPrefix = '@contact.fields.';
```
Let's try to use meaningful variable names